README.md

```diff
@@ -2,16 +2,58 @@
 
 1. Clone the repository
 
-2. Export the environment variables below
+2. Export the environment variables below (these variables below will be needed for the kitchen test inspec, not the terraform)
 
 ```
 export AWS_ACCESS_KEY_ID="<your-access-id>"           
 export AWS_SECRET_ACCESS_KEY="<your-secret-key>"
+export AWS_REGION='us-west-2'
+```
+
+3. Export the variables needed for the terraform code using TF_VAR_<variable_name> 
+
+```
+export TF_VAR_aws_access_key="<your_aws_access_key>"
+export TF_VAR_aws_secret_key="<your_aws_secret_key>"
+export TF_VAR_aws_region="us-west-2"
+export TF_VAR_aws_key_name="<your_key_name>"
 ```
-3. Change the values of a few variables in the terraform.tfvars file
 
 4. bundle install 
 
 5. bundle exec kitchen test
 
 6. bundle exec kitchen verify
+
+####################################
+###### SECOND PART - PACKER ########
+####################################
+
+1. cd packer
+
+2. /usr/local/bin/packer validate example.json
+
+3. /usr/local/bin/packer build example.json
+
+note: the image name was found by using the aws cli describe command:
+`ws ec2 describe-images --owners self amazon --region us-west-2`
+
+4. copy the ami generated by packer, we'll use it to launch our ec2 in the next step.
```

Comment:
What do we need to copy? How?

Reply:
I'll fix it so we can use data sources instead of copying the ami ID manually :)